### PR TITLE
Khai mai2016 plano tx

### PR DESCRIPTION
**Pull request by KhaiMai2016**
Hello All,

1. hey @dbiscoIntuit 
2. Just want to say hi 👍

### CODE REVIEW

File: KhaiMai2016-PlanoTX.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Plano, TX
+
+### Restaurants
```
Comment (trjnsteve) [inline KhaiMai2016-PlanoTX.md]: @KhaiMai2016 I will be in Plano next week, what is your favorite lunchtime restaurant

Comment (KhaiMai2016) [inline KhaiMai2016-PlanoTX.md]: there is a Samui Thai north of Plano. If you like Thai food

**Comment by KhaiMai2016** [comment]
@trjnsteve i've add name of the restaurants area 👍

**Comment by trjnsteve** [comment]
@KhaiMai2016 I will have to give it a try :+1: :ship: :shipit:

**Comment by dbiscoIntuit** [comment]
@KhaiMai2016 👍 , go ahead and 🚢

**Pull request merged**